clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -162,6 +162,11 @@
     public static final String REQUEST_TIMEOUT_MS_CONFIG = CommonClientConfigs.REQUEST_TIMEOUT_MS_CONFIG;
     private static final String REQUEST_TIMEOUT_MS_DOC = CommonClientConfigs.REQUEST_TIMEOUT_MS_DOC;
 
+    /** <code>interceptor.classes</code> */
+    public static final String INTERCEPTOR_CLASSES_CONFIG = "interceptor.classes";
+    public static final String INTERCEPTOR_CLASSES_DOC = "A list of classes to use as interceptors. "
+            + "Implementing the <code>ConsumerInterceptor</code> interfaces allows plugging in classes that can listen to events "
+            + "happening to the records at different points on consumer. By default, there are no interceptors.";
```

Comment:
I think you meant `interface` instead of `interfaces`. Also, the interface allows more than just `listening` to events right?